doc/api-extensions.md

```diff
@@ -2281,3 +2281,6 @@ When enabled and a suitable system is in use (requires ZFS 2.2 or higher), the Z
 
 This introduces support for the `all-projects` query parameter for the GET API calls to both `/1.0/operations` and `/1.0/operations?recursion=1`.
 This parameter allows bypassing the project name filter.
+
+## `metadata_configuration`
+Adds `GET /1.0/metadata/configuration` API endpoint to retrieve the generated YAML file containing the metadata configuration for all the cluster, instance and server options.
```

Comment:
YAML?

Reply:
Also, the API extension should describe the resulting API changes to the end user, rather than implementation.
So there's no need to mention the "file containing the metadata" - the end user doesn't need to know that there is a file at all, only that the new URL endpoint returns metadata about configuration in a certain format.